pkgs/applications/virtualization/crosvm/default.nix

```diff
@@ -0,0 +1,76 @@
+{ rustPlatform, stdenv, fetchgit, makeWrapper, minijail, binutils-unwrapped, pkgconfig, libusb1, buildPackages, libcap, runCommand, symlinkJoin }:
+
+let
+
+  arch = with stdenv.hostPlatform;
+    if isArm then "arm"
+    else if isx86_64 then "x86_64"
+    else throw "no seccomp policy files available for host platform";
+
+  # used to turn symlinks into real files because write permissions are necessary for the vendoring process
+  delink = src: runCommand src.name {
+    preferLocalBuild = true;
+    allowSubstitutes = false;
+  } ''
+    cp -prL --reflink=auto ${src} $out
```

Comment:
Is `--reflink=auto` not the default? The manpage doesn't seem to say.

Reply:
From nonofficial documentation I could find online that is not the case, and the default unpackPhase for directories uses `--reflink=auto` as well.

Reply:
Yeah you're right. `unpackPhase` does the same.
And it seems that the default of `cp` is `never`: https://github.com/coreutils/coreutils/blob/master/src/cp.c#L796